<?xml version="1.0"?>
<launch>

<param name="active" type="int" value="0"/>
<param name="des_pos_x" type="double" value="1"/>
<param name="des_pos_y" type="double" value="1"/>

<node pkg="final_assignment" type="user_interface.py" name="user_interface" required="true" launch-prefix="xterm -bg black -fg green -e"/>
<node pkg="final_assignment" type="behaviour_1.py" name="behaviour_1" required="true" launch-prefix="xterm -bg black -fg green -e"/>
<node pkg="final_assignment" type="behaviour_2.py" name="behaviour_2" required="true" launch-prefix="xterm -bg black -fg green -e"/>
<node pkg="final_assignment" type="behaviour_3.py" name="behaviour_3" required="true" launch-prefix="xterm -bg black -fg green -e"/>
</launch>
